client/src/context/useTeamBoardContext.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState, useContext, createContext, FunctionComponent } from 'react';
+
+interface ITeamBoardContext {
+  teams: any;
+  activeTeam: any;
```

Comment:
Can we add types onto these, it utilises TypeScript more and allows for less problematic type errors in the future especially for the team. 👍🏻